pkg/core/blockchain.go

```diff
@@ -1060,9 +1060,8 @@ func (bc *Blockchain) CalculateClaimable(value int64, startHeight, endHeight uin
 }
 
 // FeePerByte returns transaction network fee per byte.
-// TODO: should be implemented as part of PolicyContract
 func (bc *Blockchain) FeePerByte() util.Fixed8 {
-	return util.Fixed8(1000)
+	return util.Fixed8(bc.contracts.Policy.GetFeePerByteInternal(bc.newInteropContext(trigger.Application, bc.dao, nil, nil)))
```

Comment:
I think this one is worth adding some function that don't require creating an interop contexts. It's just a get from the DB essentially.

Reply:
Replaced `(*Policy).GetFeePerByteInternal(ic *interop.Context, ...)` with `(*Policy).GetFeePerByteInternal(dao dao.DAO, ...)`, so now we don't have to create separate interop context for this method. Same thing for `GetMaxTransactionsPerBlockInternal` and `GetBlockedAccountsInternal`